docs/excel/excel-add-ins-charts-data-labels.md

```diff
@@ -0,0 +1,230 @@
+
+- Change data label shapes by setting the [geometricShapeType](/javascript/api/excel/excel.chartdatalabel)  property.
+- Change height and width using the [setWidth and setHeight](/javascript/api/excel/excel.chartdatalabel) methods.
+- Change the text using the [text](/javascript/api/excel/excel.chartdatalabel) property.
+- Change the text formatting using the [format](/javascript/api/excel/excel.chartdatalabel) property. You can change the [border, fill, and font](/javascript/api/excel/excel.chartdatalabelformat) properties.
+
+The following code sample shows how to set the shape type, height and width, and font formatting for the data labels.
+
+```javascript
+ await Excel.run(async (context) => {
+    const worksheet = context.workbook.worksheets.getActiveWorksheet();
+    const chart = worksheet.charts.getItem("PartsChart");
+    const series = chart.series.getItemAt(5);
+
+    // Set geometric shape of data labels to cubes.
+    series.dataLabels.geometricShapeType = Excel.GeometricShapeType.cube;
+    series.points.load("count");
+    await context.sync();
+    let pointsLoaded = series.points.count;
+
+    // Change height, width, and font size of all data labels.
+    for (let j = 0; j < pointsLoaded; j++) {
+      series.points.getItemAt(j).dataLabel.setWidth(60);
+      series.points.getItemAt(j).dataLabel.setHeight(30);
+      series.points.getItemAt(j).dataLabel.format.font.size = 12;
+    }
+
+    // Set text of a data label.
+    series.points.getItemAt(2).dataLabel.setWidth(80);
+    series.points.getItemAt(2).dataLabel.setHeight(50);
+    series.points.getItemAt(2).dataLabel.text = "Spokes Qtr3: 4589 ↓";
+
+    await context.sync();
+  });
+```
+
+:::image type="content" source="../images/excel-data-labels-chart-formats.png" alt-text="Screenshot of chart with data labels set to cubes, new size, and custom text in one of the data labels.":::
+
+You can also change the formatting of text in a data label. The following code sample shows how to use the [getSubstring](/javascript/api/excel/excel.chartdatalabel) method to get part of data label text and apply font formatting.
+
+```javascript
+await Excel.run(async (context) => {
+    const worksheet = context.workbook.worksheets.getActiveWorksheet();
+    const chart = worksheet.charts.getItem("PartsChart");
+    const series = chart.series.getItemAt(5);
+
+    // Get the "Spokes" data label.
+    let label = series.points.getItemAt(2).dataLabel;
+    label.load();
+    await context.sync();
+
+    // Change border weight of this label.
+    label.format.border.weight = 2;
+    // Format "Qtr3" as bold and italicized. 
+    label.getSubstring(7, 4).font.bold = true;
+    label.getSubstring(7, 4).font.italic = true;
+    // Format "Spokes" as green.
+    label.getSubstring(0, 6).font.color = "green";
+    // Format "4589" as red.
+    label.getSubstring(12).font.color = "red";
+    // Move label up by 15 points.
+    label.top = label.top - 15;
+
+    await context.sync();
+```
+
```

Comment:
Is there something amiss in the previous code sample? Because starting from this point, the syntax highlighting is broken. Ie. Image markdown should be white, H2 is azure blue, and regular paras are white. Everything is light blue from here on out. It looks correct on the review site, but I'm curious what's causing it and if anything will break when published.

Reply:
I made some changes, so hopefully fixed. build looks okay.